backend/papersfeed/utils/collections/utils.py

```diff
@@ -291,21 +292,47 @@ def __get_collections_contains_paper(paper_id, request_user):
     return [collection.id for collection in collections]
 
 
-def __remove_paper_from_collections(paper_id, collection_ids):
+def __remove_paper_from_collections(paper_id, collection_ids, request_user):
     """Remove Paper From Collections"""
     if collection_ids:
         CollectionPaper.objects.filter(
             collection_id__in=collection_ids, paper_id=paper_id
         ).delete()
 
+    # Update action count for recommendation
+    for _ in range(len(collection_ids)):
```

Comment:
Good

Reply:
Error occurs when I use list.Count(because it is not a queryset, but just a list). Also, count() function does different thing from what I expect (it needs arg and count the number of arg in the list).ㅠㅠ

Reply:
Oh, sorry. I misunderstood this code.

Reply:
By the way, `QuerySet.count()` usage is actually same with `len`.

Reply:
@kosumin The link 569 of papers/utils.py of the current master has the usage I mentioned about QuerySet.

Reply:
Thank you! I would remember that!